src/app/app.component.html

```diff
@@ -1,3 +1,3 @@
 <ion-app>
-  <ion-router-outlet></ion-router-outlet>
+  <app-menu></app-menu>
```

Comment:
Se non volessimo utilizzare il menu in tutte le pages dell'app? il componente `app-menu` va importato solamente nella **home page** per adesso.

Reply:
done